### PR TITLE
Add an e2e test for snowpack

### PR DESCRIPTION
**Pull request by alubbe**
I added an e2e test for snowpack to catch issues like https://github.com/yarnpkg/berry/issues/2229 earlier.

I also validated locally that this new e2e test fails against 2.4.0, but passes against the current master.

### CODE REVIEW

**Comment by alubbe** [comment]
@arcanis is there any chance you could merge this soon? I keep getting 30+ emails per day about CI failures that this branch didn't cause :D

File: .github/workflows/e2e-snowpack-workflow.yml [hunk 1]
```diff
@@ -37,4 +37,4 @@ jobs:
       run: |
         source scripts/e2e-setup-ci.sh
         yarn dlx create-snowpack-app my-csa-ts --template @snowpack/app-template-react-typescript --use-yarn
-        cd my-csa-ts && yarn build
+        cd my-csa-ts && yarn add @types/mocha && yarn build
```
Comment (merceyz) [inline .github/workflows/e2e-snowpack-workflow.yml:40]: Shouldn't this be added upstream?

**Comment by merceyz** [comment]
> I keep getting 30+ emails per day about CI failures that this branch didn't cause :D

You have actions enabled on your fork so you're running our e2e tests every four hours, might want to turn that off https://github.com/alubbe/berry/actions

**Comment by arcanis** [comment]
Perfect, thanks @alubbe!

**Comment by alubbe** [comment]
> > I keep getting 30+ emails per day about CI failures that this branch didn't cause :D
> 
> You have actions enabled on your fork so you're running our e2e tests every four hours, might want to turn that off https://github.com/alubbe/berry/actions

Thanks that helps my inbox a lot! :D

**Comment by arcanis** [comment]
Ironically it seems we merged it just in time to spot a regression from 3.0.3 to 3.0.4 😅

https://github.com/yarnpkg/berry/actions?query=workflow%3A%22E2E+Snowpack%22

**Pull request merged**